Testcase4-Application-breakdown/alexa/src/infra/model/BuiltinSlotTypes.js

```diff
@@ -1,3 +1,15 @@
+/*
+ * Copyright (c) 2020 Institution of Parallel and Distributed System, Shanghai Jiao Tong University
+ * ServerlessBench is licensed under the Mulan PSL v1.
+ * You can use this software according to the terms and conditions of the Mulan PSL v1.
+ * You may obtain a copy of Mulan PSL v1 at:
+ *     http://license.coscl.org.cn/MulanPSL
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT, MERCHANTABILITY OR FIT FOR A PARTICULAR
+ * PURPOSE.
+ * See the Mulan PSL v1 for more details.
+ */
+
 //import {ISlotValue, SlotMatch, SlotType} from "virtual-core";
```

Comment:
please remove this comment (it's from the original project which is in typescript)